addon/components/ember-flatpickr.js

```diff
@@ -72,7 +74,7 @@ export default class EmberFlatpickr extends Component {
 
   @action
   onWillDestroy() {
-    this.field._flatpickr.destroy();
```

Comment:
Does `_flatpickr` no longer work?

Reply:
No it works. I've only changed the fetching of `this.field._flatpickr` to a getter property.